packages/pwa/app/components/cart-item-variant/index.jsx

```diff
@@ -9,30 +9,25 @@ import PropTypes from 'prop-types'
 
 /**
  * This component and associated context/hook provide a convenient wrapper
- * around a group of components used for rendering product variant details
- * from items in a customer basket or order.
+ * around a group of components used for rendering product variant details.
  */
 
-const CartItemVariantContext = createContext()
+const ItemVariantContext = createContext()
```

Comment:
This file has the majority of the changes in this PR, the rest of this PR is just find/replace.